### ISSUE TITLE
Is this library still being developed?

### ISSUE THREAD

**Issue by ErcinDedeoglu** [CreateJS/CreateJS#7]
Is this library still being developed?

**Comment by owendwyer** [comment]
Hi,

See this thread https://github.com/CreateJS/EaselJS/issues/1040

And this Slack https://app.slack.com/client/TU5SGL79R/CUHBD3DM3

**Comment by Pi-Bouf** [comment]
@owendwyer your slack's link is broken :D

**Comment by owendwyer** [comment]
Yes, it seems the Slack channel isn't working anymore.  I guess recent events have changed people's priorities.

**Comment by nicolasgoudard** [comment]
So I have the same question, I don't understand the consortium because I don't speak english very well... I hope this project is still supported. So is it still supported or not ? Because there are not updates since 2 years. I use EaselJS for all my project, because it offers high levels functionnalities to manipulate shapes, like Java AWT.  I converted java programs to JS + EaselJS + HammerJS, so I hope I can keep this library

**Comment by nicolasgoudard** [comment]
I have selected this project because I saw it is sponsored by Adobe and Microsoft.... So why is there a silent since 2 years ?

**Comment by nicolasgoudard** [comment]
I see another project : FabricJS but I don' t like it too much but I see it is active. why this silent for EaselJS ???
Nobody needs high level functionalities for JS games in canvas  ???

**Comment by dijitaletkinlikler** [comment]
It is maintained with zimjs by Danzen et al. Because Zimjs is built on createjs, danzen takes care of createjs related issues. You can check this slack group.

https://zimjs.slack.com/
